.github/workflows/ci.yml

```diff
@@ -417,21 +431,21 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dtolnay/rust-toolchain@v1
         with:
-          toolchain: stable
-          components: clippy
+          toolchain: nightly
+          components: clippy, rust-src
       - uses: Swatinem/rust-cache@v2
 
       # Run clippy on all packages targeting RISC-V.
       - name: clippy (esp-riscv-rt)
-        run: cargo +stable clippy --manifest-path=esp-riscv-rt/Cargo.toml -- --no-deps
+        run: cd esp-riscv-rt/ && cargo +nightly clippy -- --no-deps
```

Comment:
Why are these tested on nightly now? AFAIK we don't use any nightly features inside esp-riscv-rt?

Reply:
This might have been a consistency/copy-paste thingy. I don't quite remember which one, but either way using a single toolchain looks better and using nightly means usually more lints.